### DISCUSSION TITLE
How to handle missing jpeg2000 support?

### DISCUSSION THREAD

**Discussion by volschin** [immich-app/immich#14427] [Q&A]
I‘m trying out immich and have seen on my test import of about 20K files from iCloud that .jp2 files could not be imported. Immich doesn‘t support this format. Are there any suggestions how to handle this the best way without quality loss and not too much memory expansion?

**Comment by mertalev** [comment]
JPEG2000 is not a supported format, but our image processing should be able to handle it just fine. We could potentially add it as a supported format.

In the meantime, you can try changing the extension to jpg for an asset and uploading it. Immich will accept the asset, but you should also check that the thumbnail is created successfully. If it is, you can do the same with the other assets and upload them as "JPEG" files.

**Comment by mmomjian** [comment]
Now supported
